src/dvclive/live.py

```diff
@@ -434,7 +450,14 @@ def log_plot(
         plot.dump(datapoints)
         logger.debug(f"Logged {name}")
 
-    def log_sklearn_plot(self, kind, labels, predictions, name=None, **kwargs):
+    def log_sklearn_plot(
+        self,
+        kind: Literal["calibration", "confusion_matrix", "precision_recall", "roc"],
```

Comment:
I think it's missing `"det"`? 

Also, can we import from here:

https://github.com/iterative/dvclive/blob/38f2f51c300a847e60894eff47f65fc36e4a9a25/src/dvclive/plots/__init__.py#L7-L13

Reply:
I see it's missing in the docs also

Reply:
https://github.com/iterative/dvc.org/pull/5103